README.rst

```diff
@@ -309,6 +309,9 @@ editing:
 	sops:
 	    kms:
 	    - arn: arn:aws:kms:us-east-1:656532927350:key/920aff2e-c5f1-4040-943a-047fa387b27e
+          aws_profile: foo
+
+If no aws_profile is specified `default` is used for KMS.
```

Comment:
I'd word this the other way around. Most people aren't going to use profiles, so I'd say something like:

```
If you want to use a specific profile, you can do so with `aws_profile`:

... example using aws_profile ...

```

In general, I just wouldn't include `aws_profile` in every single example. A brief explanation in the section about AWS KMS is more than enough, because I feel having it everywhere will confuse people.